scrap/api.py

```
@@ -0,0 +1,51 @@
+from requests import get
+from dotenv import load_dotenv
+import os
+import math
+
+load_dotenv()
+PAGE_SIZE = 100
+MAX_START = 1000
+
+
+def fetch_api(query):
+    URL = os.getenv("NAVER_API_URL")
```

Pylint (3.3.7)

[convention] Variable name "URL" doesn't conform to snake_case naming style

(C0103)